### PR TITLE
Removed xref, added steps for editing VM yaml

### PR DESCRIPTION
**Pull request by apinnick**

### CODE REVIEW

File: modules/cnv-importing-vmware-vm.adoc [hunk 1]
```diff
@@ -70,8 +70,11 @@ If you do not select a storage class, {CNVProductName} uses the default storage
 
 . In the *Advanced* screen, enter the *Hostname* and *Authorized SSH Keys* if you are using `cloud-init`.
 +
-You must xref:../../../cnv/cnv_virtual_machines/cnv-edit-vms.adoc#cnv-editing-vm-yaml-web_cnv-edit-vms[edit the virtual machine's YAML configuration] after importing it:
+You must edit the virtual machine's YAML configuration after importing it:
 
+.. Click *Workloads* -> *Virtual Machine* from the side menu.
+.. Select the virtual machine you imported.
```
Comment (kalexand-rh) [inline modules/cnv-importing-vmware-vm.adoc]: I'd s/you/that you

> Do not omit the word that from clauses.

**Review by kalexand-rh — COMMENTED** [review]
One pick and then it LGTM

**Comment by apinnick** [comment]
@kalexand-rh  Fixed.

**Pull request merged**